source/docs/github-pages.md

```diff
@@ -2,39 +2,55 @@
 title: GitHub Pages
 ---
 
-In this tutorial, we use [Travis CI](https://travis-ci.com/) to deploy Github Pages. It is free for open source repository, meaning your repository's `master` branch has to be public. Please skip to the [Private repository](#Private-repository) section if you prefer to keep the repo private, or prefer not to upload your source folder to GitHub.
+In this tutorial, we use [GitHub Actions](https://docs.github.com/en/actions) to deploy Github Pages. It works in both public and private repository. Skip to the [One-command deployment](#One-command-deployment) section if you prefer not to upload your source folder to GitHub.
 
 1. Create a repo named <b>*username*.github.io</b>, where username is your username on GitHub. If you have already uploaded to other repo, rename the repo instead.
-2. Push the files of your Hexo folder to the repository. The `public/` folder is not (and should not be) uploaded by default, make sure the `.gitignore` file contains `public/` line. The folder structure should be roughly similar to [this repo](https://github.com/hexojs/hexo-starter), without the `.gitmodules` file.
-3. Add [Travis CI](https://github.com/marketplace/travis-ci) to your account.
-4. Go to [Applications settings](https://github.com/settings/installations), configure Travis CI to have access to the repo.
-5. You'll be redirected to Travis page.
-6. On a new tab, generate a [new token](https://github.com/settings/tokens) with **repo** scopes. Note down the token value.
-7. On the Travis page, go to your repo's setting. Under **Environment Variables**, put **GH_TOKEN** as name and paste the token onto value. Click Add to save it.
-8. Add `.travis.yml` file to your repo (alongside _config.yml & package.json) with the following content:
-```yml
-sudo: false
-language: node_js
-node_js:
-  - 10 # use nodejs v10 LTS
-cache: npm
-branches:
-  only:
-    - master # build master branch only
-script:
-  - hexo generate # generate static files
-deploy:
-  provider: pages
-  skip-cleanup: true
-  github-token: $GH_TOKEN
-  keep-history: true
-  on:
-    branch: master
-  local-dir: public
+2. Push the files of your Hexo folder to the **`source` branch** of your repository. The `public/` folder is not (and should not be) uploaded by default, make sure the `.gitignore` file contains `public/` line. The folder structure should be roughly similar to [this repo](https://github.com/hexojs/hexo-starter), without the `.gitmodules` file.
+  - To push to `source` branch:
+  ```
+  $ git push origin source
+  ```
+3. Add `.github/workflows/pages.yml` file to your repo with the following content:
+
+```yml .github/workflows/pages.yml
+name: Pages
+
+on:
+  push:
+    branches:
+      - source  # default branch
```

Comment:
I haven't test this, seems strange to me. https://github.com/marketplace/actions/github-pages-action#%EF%B8%8F-repository-type---user-and-organization

Reply:
FYI, I am using `peaceiris/actions-gh-pages@v2` which WFM.

Reply:
Are you using master+gh-pages (most common?) or source+master?

Reply:
@curbengh 

`master` & `gh-pages`. While `peaceiris/actions-gh-pages@v2` supports both of them.

Also, GitHub is testing publishing github pages from any branch, so it won't matter in the future.

Reply:
> While `peaceiris/actions-gh-pages@v2` supports both of them.

I find it puzzling the [docs](https://github.com/marketplace/actions/github-pages-action#%EF%B8%8F-repository-type---user-and-organization) suggest switching to `source+master`, while `master+gh-pages` still works. Should I retain `source+master` in this PR?

Reply:
> Should I retain `source+master` in this PR?

It would be fine.

Reply:
Looks like `User page` repo requires `source+master`. https://github.com/hexojs/site/issues/1324